### ISSUE TITLE
Bug: `forge fmt` output fails on `forge fmt --check`

### ISSUE THREAD

**Issue by xJonathanLEI** [foundry-rs/foundry#2189]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (24a9fb8 2022-07-02T00:03:35.337245982Z)

### What command(s) is the bug in?

forge fmt

### Operating System

Linux

## Description

Formatted output from `forge fmt` is still considered not formatted when using `--check`, aka. the same input can be formatted twice.

## Steps to reproduce

Format this code:

```solidity
// SPDX-License-Identifier: MIT
pragma solidity =0.8.15;

contract Format {
    constructor() {
        uint256 a = 1;

        // TODO: do this and that
    }
}
```

and get this result:

```solidity
// SPDX-License-Identifier: MIT
pragma solidity =0.8.15;

contract Format {
    constructor() {
        uint256 a = 1;
        // TODO: do this and that
    }
}
```

Run `format fmt` again.

## Expected result

The file stays unchanged since it's already formatted.

## Actual result

The file gets formatted again (and thus fails if `--check` is used):

```solidity
// SPDX-License-Identifier: MIT
pragma solidity =0.8.15;

contract Format {
    constructor() {
        uint256 a = 1; // TODO: do this and that
    }
}
```

**Comment by rkrasiuk** [comment]
@xJonathanLEI can we consider this issue closed?

**Comment by xJonathanLEI** [comment]
@rkrasiuk Yep I've tested and it's fixed by #2197.
